.github/workflows/docker-tag-v1.yml

```diff
@@ -0,0 +1,45 @@
+---
+name: Docker tag
+
+on:
+  workflow_call:
+    inputs:
+      aws_region:
+        type: string
+        default: "eu-central-1"
+      ci_iam_role:
+        type: string
+        default: ""
```

Comment:
I would move the optional inputs parameters after the required one, this is usually a good practice to ease readability / maintainability (IMHO)

Reply:
ok I don't have a strong opinion on that
most of our workflows parameters are sorted in alphabetical order
I think that's a good suggestion but we need to that consistently in all workflows

Reply:
I have created a dedicated card for this:
https://www.notion.so/sencrop/Reorder-parameters-on-Github-workflows-63fd0e85e1294399958faeb287fedc27?pvs=4